packages/cli-hydrogen/src/cli/services/deploy.ts

```diff
@@ -16,9 +16,19 @@ export async function deployToOxygen(_config: DeployConfig) {
 
   // note: need to handle this error
   const buildResponse = await runBuildCommandStep(config, assetBaseURL)
-  const deployResponse = await uploadDeploymentStep(config, deploymentID)
+  const previewURL = await uploadDeploymentStep(config, deploymentID)
 
-  output.success('Deployed and healthy!')
-  output.info(`• Preview URL: ${deployResponse}`)
-  // note: should try output URL after the "success" message for scripting purposes like oxygenctl does right now
+  output.info(`Preview URL: ${previewURL}`)
+
+  if (!config.skipHealthCheck) {
+    const isHealthy = await healthCheck(previewURL)
+    if (!isHealthy) {
+      output.info("🤕 The deployment was uploaded but can't be reached yet.")
+      return
+    }
+    output.success('✅ Deployed and healthy!')
+  }
+
+  // For scripting purposes, last line is preview URL.
+  output.info(previewURL)
```

Comment:
Does this show up after the final `✨  Done in 2.20s.` message? When I was testing previously it did not IIRC